pydis_site/README.md

```diff
@@ -0,0 +1,52 @@
+# `pydis_site` project directory
+
+This directory hosts the root of our **Django project**[^1], and is responsible
+for all logic powering our website. Let's go over the directories in detail:
+
+- [`apps`](./apps) contains our **Django apps**. If you want to add your own API
+  endpoint or new functionality to our homepage, that's the place to go.
+  <!-- Each individual application also has its own README.md that you can click
+  through. -->
```

Comment:
Why is this comment here if it's not visible when rendered?

Reply:
It's a placeholder for the documentation I plan to add later (#674), so it's commented out for when that has a PR. Or should we already add it now?

Reply:
d5c3d0b3